internal/service/service.go

```diff
@@ -150,10 +150,16 @@ func (s *Service) decryptDriverData(key string, val interface{}) (ret interface{
 		return "ENC[" + parts[1] + "]", true
 	}
 
-	data := []byte(parts[1])
-	decoded, err := base64.StdEncoding.DecodeString(string(data))
-	if err != nil {
-		dipper.Logger.Panicf("encrypted data shoud be base64 encoded")
+	var decoded []byte
+
+	if parts[1][0] == '"' {
+		decoded = []byte(strings.Trim(parts[1], "\""))
+	} else {
+		var err error
+		decoded, err = base64.StdEncoding.DecodeString(parts[1])
+		if err != nil {
+			dipper.Logger.Panicf("encrypted data shoud be base64 encoded")
```

Comment:
minor typo (shoud vs should) (exists in the source and the other place this error exists as well)

Reply:
Fixed.